pkg/core/native/crypto.go

```diff
@@ -134,6 +146,11 @@ func newCrypto() *Crypto {
 		manifest.NewParameter("signature", smartcontract.ByteArrayType))
 	md = NewMethodAndPrice(c.recoverSecp256K1, 1<<15, callflag.NoneFlag, config.HFEchidna)
 	c.AddMethod(md, desc)
+
+	desc = NewDescriptor("bls12381MultiExp", smartcontract.InteropInterfaceType,
+		manifest.NewParameter("pairs", smartcontract.ArrayType))
+	md = NewMethodAndPrice(c.bls12381MultiExp, 1<<23, callflag.NoneFlag, config.HFFaun)
+	c.AddMethod(md, desc)
```

Comment:
Declare Ethereum-compatible serializators in the contract manifest, add compatibility tests for these APIs to ensure that the combination of [deserialize(p), do some math(p), serialize(p)] gives proper results.